### ISSUE TITLE
Display contacts only from enabled addressbooks

### ISSUE THREAD

**Issue by efelon** [nextcloud/maps#169]
Nextcloud 16.0.4
Maps 0.1.2

Maps App should only displays contacts from enabled address books. At the moment it show also disabled ones. 
This helps to "bulk-hide" contacts now, since there is no additional filtering in the app yet, but also in general imho.

Best regards
efelon

**Comment by jancborchardt** [comment]
Yup, this should basically simply mirror what is enabled or not in the Contacts app. No additional interface options needed.

**Comment by eneiluj** [comment]
Yep I agree. I'd like to only show contacts from enabled address books.

@skjnldsv Could you give me a hint about how to know if an address book is enabled or not in Contacts app from another app? I get the address book objects from NC contacts manager and also directly from the cardDav backend but I didn't find out how to know which address book is enabled.

**Comment by skjnldsv** [comment]
There is a dedicated prop for it.
No idea how you can get it in php, maybe it's straightforward? cc @georgehrke

**Comment by georgehrke** [comment]
Looking at https://github.com/nextcloud/server/blob/master/apps/dav/lib/CardDAV/AddressBookImpl.php I would say that it's not exposed at the moment.

**Comment by eneiluj** [comment]
So how does the Contacts app know which one is enabled?

**Comment by georgehrke** [comment]
It uses its own CardDAV property which is stored inside oc_properties:

![19E96B59-2EA8-4562-BFC2-76DDD25F0EA9](https://user-images.githubusercontent.com/1250540/66760384-46b6ab80-eea2-11e9-9de1-2bac18e9b58c.png)

**Comment by eneiluj** [comment]
Yay thank you.
